h2o-core/src/main/java/water/api/FramesHandler.java

```diff
@@ -69,7 +69,7 @@
         // Weed out frames with vecs that are no longer in DKV
         boolean skip = false;
         for( Vec vec : frame.vecs() ) {
-          if (DKV.get(vec._key) == null) {
+          if (vec == null || DKV.get(vec._key) == null) {
```

Comment:
@ashrith was seeing this issue with 'getFrames' in Flow, would NPE and show nothing.